locations/spiders/speedway_us.py

```diff
@@ -1,73 +1,49 @@
+            "Content-Type": "application/json",
+            "Accept": "application/json, text/plain, */*",
+        }
+
+        return JsonRequest(url=url, method="POST", headers=headers, data=data, meta=dict(offset=offset, limit=limit))
 
     def start_requests(self) -> Iterable[Request]:
         yield self.make_request(0)
 
     def parse(self, response: Response, **kwargs: Any) -> Any:
-        for location in response.xpath('//section[@class="c-location-card"]'):
-            item = Feature()
-            item["ref"] = location.xpath("@data-costcenter").get()
-            item["lat"] = location.xpath("@data-latitude").get()
-            item["lon"] = location.xpath("@data-longitude").get()
-            item["state"] = location.xpath("@data-state").get()
-            item["website"] = response.urljoin(location.xpath(".//a/@href").get())
-            item["addr_full"] = location.xpath(".//@data-location-address").get()
-            item["street_address"] = location.xpath('.//h2[@data-location-info-window-data="address"]/text()').get()
-            item["phone"] = location.xpath('.//li[data-location-details="phone"]/text()').get()
-
-            amenities = location.xpath(
-                './/div[@class="c-location-options--amenities"]/ul[@class="c-location-options__list"]/li/span/text()'
-            ).getall()
-            fuels = location.xpath(
-                './/div[@class="c-location-options--fuel"]/ul[@class="c-location-options__list"]/li/span/text()'
-            ).getall()
-
-            if "Open 24 Hours" in amenities:
-                item["opening_hours"] = "24/7"
-
-            # TODO: "Money Orders" "Propane Exchange"
-            apply_yes_no(Extras.ATM, item, "ATM" in amenities)
-            apply_yes_no(Extras.CAR_WASH, item, "Car Wash" in amenities)
-            apply_yes_no(Fuel.PROPANE, item, "Propane Exchange" in amenities)
-            # TODO: DEF Plus Premium Unleaded
-            apply_yes_no(Fuel.DIESEL, item, "Auto Diesel" in fuels)
-            apply_yes_no(Fuel.E85, item, "E-85" in fuels)
-            apply_yes_no(Fuel.E15, item, "E15" in fuels)
-            apply_yes_no(Fuel.E20, item, "E20" in fuels)
-            apply_yes_no(Fuel.E30, item, "E30" in fuels)
-            apply_yes_no(Fuel.ETHANOL_FREE, item, "Ethanol Free" in fuels)
-            apply_yes_no(Fuel.KEROSENE, item, "Kerosene" in fuels)
-            apply_yes_no(Fuel.OCTANE_90, item, "Plus 90" in fuels)
-            apply_yes_no(Fuel.OCTANE_91, item, "Plus 91" in fuels)
-
+        for store in response.json()["data"]["stores"]:
+            item = DictParser.parse(store)
+            item["street_address"] = item.pop("addr_full")
+            item["website"] = "https://www.speedway.com/"
```

Comment:
Please try to find a website specific to each location rather than manually setting it to the generic website for the whole brand.

Reply:
Couldn't find any website specific to POI location.